root/etc/e-smith/templates/etc/shells/10base

```diff
@@ -0,0 +1,6 @@
+/bin/bash
```

Comment:
I'd like to move lines from 1 to 6 in a fragment named something like "10base"  or "10default".

And the line 6 to a fragment named something like "20nethserver"